libs/utils/include/celix_errno.h

```diff
@@ -61,6 +73,44 @@ typedef int celix_status_t;
  */
 const char* celix_strerror(celix_status_t status);
 
+/*!
+ * Customer error code mask
+ *
+ */
+#define CELIX_CUSTOMER_ERR_MASK 0x20000000
+
+/*!
+ * The facility of system error code,
+ * \note Error code 0 indicates success,it is not system error code.
+ */
+#define CELIX_FACILITY_SYSTEM 0
```

Comment:
De we want a facility `0` code or always ensure a not `0` to more easily see of its a valid error code.

Reply:
I whould like facility 0  indicate C errno. Because some interfaces has treat C errno as their return value.   for example `celixThreadMutex_create,celixThreadMutex_lock`,etc.

Reply:
Good point